### DISCUSSION TITLE
Is there any website where I can post figma design?

### DISCUSSION THREAD

**Discussion by shahab570** [codesandbox/codesandbox-client#5286] [General]
Usually, I make various designs and post them to codepen website. I want to know that is there any website where I can post my Figma design project?

**Comment by SaraVieira** [comment]
hey!

Sorry I don't think I don't understand what you mean

**Reply by shahab570** [reply]
Graphics designers use Behance to show their design to the people. Programmers use GitHub to show their code. IF I want to show my Figma projects to people, how can I do so?

**Comment by suczker** [comment]
Do you mean creating your own template?

**Reply by shahab570** [reply]
NO. I  mean how can I share my Figma project with a larger audience
